packages/popover/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+<a name="0.2.0-alpha.c35333c6"></a>
+# 0.2.0-alpha.c35333c6 (2018-06-15)
+
+
+### Features
+
+* **popover:** initial scaffold ([03e43dc](https://github.com/CraveFood/farmblocks/commit/03e43dc)), closes [#392](https://github.com/CraveFood/farmblocks/issues/392)
+* **Popover:** fully functional component ([eb244a6](https://github.com/CraveFood/farmblocks/commit/eb244a6))
```

Comment:
I am assuming that this got generated because you had to manually publish the package, right?

Reply:
yes... I shouldn't have commited but...